### ISSUE TITLE
Add support for other page sizes

### ISSUE THREAD

**Issue by rg-wood** [rgladwell/ink-elements#31]
Currently we only support the A4 page size.

Using the `page-size` attribute on the `ink-doc` element you can specify page size.

For example:

```html
<ink-doc page-size="letter">
  <ink-page>
    <h1>letter</h1>
  </ink-page>
</ink-doc>
```

**Comment by rg-wood** [comment]
Or should this be a CSS property?

**Comment by rg-wood** [comment]
Closed by commit d0550caf4c579faeb55066c9b7c247e9ea6af9ab
